recipes/emmix/1.3/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: emmix
+  version: "1.3"
+
+source:
+  fn: EMMIX.f
+  url: https://people.smp.uq.edu.au/GeoffMcLachlan/emmix/EMMIX.f
+  md5: 66523b6873e820bde7561182dd522898
+
+build:
+  skip: True  #[osx]
+  number: 0
+
+requirements:
+  build:
+    - certifi
+    - gcc # [linux]
+
+test:
+  # EMMIX is interactive (requiring user input), so cannot be tested.
+
+about:
+  home: 'https://people.smp.uq.edu.au/GeoffMcLachlan/emmix/emmix.html'
+  summary: 'A tool that fits a mixture model of multivariate normal or t-distributed components to a given data set.'
+  license: 'Available freely for non-commercial use only'
```

Comment:
Let's put is this way, in theory you need somehow to ensure that your users know about this restriction ...